### ISSUE TITLE
a small typo in your document

### ISSUE THREAD

**Issue by whalehole** [mdn/kuma#7586]
# Content Issue

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# URL of page you are seeing the problem on:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice

# Details

<img width="781" alt="08bab3ddff60a3ab19ff266e4b37e2da" src="https://user-images.githubusercontent.com/68460417/97103514-efb7f900-16e7-11eb-9483-f11fb47be2b6.png">

**Comment by chrisdavidmills** [comment]
@whalehole I feel stupid now; what is the typo you are referring to? ; -)

**Comment by whalehole** [comment]
Haha xD shouldn’t it be fruits.slice(1,2) instead?

**Comment by chrisdavidmills** [comment]
@whalehole Ah, I see what you are referring to. The answer is no — it shouldn't be `1,2`; `1,3` is correct with what the comments are saying the output is.

The first argument is the index number of the first array element in the slice. But the second argument is the index number of the array element _after_ the point where you want to stop slicing, so the one after the last array element in the slice.

To put it another way the array slice is inclusive of the first specified element, but _not_ inclusive of the last one. Try it, and you'll see. I just tested it in Chrome and Firefox, and that's how it works. I'm not convinced this is the most logical way for it to work, but that's what they went with.
